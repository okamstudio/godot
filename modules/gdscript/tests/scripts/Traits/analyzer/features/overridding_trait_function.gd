trait SomeTrait:
	func some_func(param: Node2D)

class SomeClass:
	uses SomeTrait

	func some_func(_param: Node):
		print("overridden some func")

func test():
	var obj := Node.new()
	SomeClass.new().some_func(obj)
	obj.free()
	print("ok")
