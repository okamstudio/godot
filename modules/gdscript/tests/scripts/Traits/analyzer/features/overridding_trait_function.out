GDTEST_OK
overridden some func
ok
